src/parameter_declarations.ts

```diff
@@ -340,3 +342,10 @@ Parameter.declare({
 		'(OPTIONAL) Derives a drive key for the given drive ID without verifying its correctness against the drive on chain.',
 	type: 'boolean'
 });
+
+Parameter.declare({
+	name: NoBundleParameter,
+	aliases: ['--no-bundle'],
```

Comment:
I really tried to break the usage of this parameter but it works really well!
I tried 
```shell
$ ardrive create-drive --no-no-bundle
$ ardrive create-drive --bundle
```
And both says 'unknwon command'